packages/core/content-type-builder/admin/src/components/DataManagerProvider/index.js

```diff
@@ -569,20 +570,31 @@ const DataManagerProvider = ({
           {isInDevelopmentMode && (
             <>
               <FormModal />
-              <PopUpWarning
-                isOpen={infoModals}
-                toggleModal={toggleModalCancel}
-                content={{
-                  message: getTrad(
+              {/* TO FIX - not sure didModifiedComponents return a valuable information */}
+              <ConfirmDialog
+                bodyText={{
+                  id: getTrad(
                     `popUpWarning.bodyMessage.cancel-modifications${
                       didModifiedComponents ? '.with-components' : ''
                     }`
                   ),
+                  defaultMessage: `Are you sure you want to cancel your modifications? ${didModifiedComponents &&
+                    'Some components have been created or modified...'}`,
                 }}
-                popUpWarningType="danger"
+                isOpen={infoModals}
+                onToggleDialog={toggleModalCancel}
                 onConfirm={() => {
                   cancelChanges();
                 }}
+                iconRightButton={<CheckIcon />}
+                title={{
+                  id: getTrad('app.components.ConfirmDialog.title'),
+                  defaultMessage: 'Confirmation',
+                }}
+                rightButtonText={{
+                  id: getTrad('app.components.Button.confirm'),
```

Comment:
You don't need the getTrad here the translations already exist in the admin package.